qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/BackupActuator.java

```diff
@@ -77,9 +80,9 @@ static BackupActuator of(final String endpoint) {
    *
    * @throws feign.FeignException if the request is not successful (e.g. 4xx or 5xx)
    */
-  @RequestLine("POST /{id}")
+  @RequestLine("POST /")
   @Headers({"Content-Type: application/json", "Accept: application/json"})
-  TakeBackupResponse take(@Param final long id);
+  TakeBackupResponse take(@RequestBody final TakeBackupRequest request);
```

Comment:
:wrench: Apparently it's also possible to use a body template: https://github.com/OpenFeign/feign#body-templates
Since the JSON is simple enough, this might be preferable over adding a dependency on spring.

Reply:
:+1: Thanks.